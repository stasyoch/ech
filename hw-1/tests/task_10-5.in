0 9 0.01 0.000001
