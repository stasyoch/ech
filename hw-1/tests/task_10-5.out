0
9
0.010000
0.000001
